### PR TITLE
database created

### PR DESCRIPTION
**Pull request by Manas-wagh**
database created

### CODE REVIEW

File: src/db/models/status.enum.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export enum Status {
```
Comment (akashg3627) [inline src/db/models/status.enum.ts:1]: make all enum capital ie eg. INTERVIEW_COMPLETED = "INTERVIEW_COMPLETED"

File: src/db/models/jobEligibility.enum.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export enum Course {
```
Comment (akashg3627) [inline src/db/models/jobEligibility.enum.ts:1]: make all enum capital ie eg. INTERVIEW_COMPLETED = "INTERVIEW_COMPLETED"

File: src/db/models/season.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  @Column({
+    primaryKey: true,
+    allowNull: false,
+    type: sequelize.UUID,
+    defaultValue: sequelize.UUIDV4,
+  })
+  id: typeof randomUUID;
+
+  @Column
+  name: string;
```
Comment (akashg3627) [inline src/db/models/season.ts:18]: Can we make the name field nullable: false so that the season name is mandatory to be filled?

Comment (Tanwish2003) [reply src/db/models/season.ts:18]: Yes , we will get that changed

Comment (Manas-wagh) [reply src/db/models/season.ts:18]: done

File: src/db/models/job.ts [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    stipend: number | null;
+  };
+
+  @Column({ defaultValue: false })
+  core: boolean;
+
+  @Column({ defaultValue: true })
+  domestic: boolean;
+
+  @Column({ defaultValue: '' })
```
[the remark this answers is not in the document]
Comment (Manas-wagh) [reply src/db/models/job.ts:109]: done

File: src/db/models/status.enum.ts (hunk 1 above)
Comment (Manas-wagh) [reply src/db/models/status.enum.ts:1]: done

**Pull request merged**